### PR TITLE
Edwin branch pull request

### PR DESCRIPTION
**Pull request by ehlangework**
Please review my amazing changes - there are no bugs of course.

### CODE REVIEW

File: constants.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#define AVALUE (1)
+#define BVALUE (2)
```
Comment (scapri) [inline constants.h:2]: Please make these names more user friendly

**Review by scapri — CHANGES_REQUESTED** [review]
Yes, amazing, no bugs, but please change those 2 constant names

File: constants.h (hunk 1 above)
Comment (ehlangework) [reply constants.h:2]: My naming conventions could use some improvement.

Comment (ehlangework) [reply constants.h:2]: I will update in the next pull request as we are already in QA test.

**Pull request merged**